docs/principles-for-package-repository-security.md

```diff
@@ -13,6 +13,18 @@ We include the below taxonomy because not all security advice applies to all pac
 
 The roadmap of security capabilities can then be used by package repositories to assess gaps, put together fundable improvement lists ([like Python Packaging WG](https://github.com/psf/fundable-packaging-improvements/blob/master/FUNDABLES.md)), or write specific grant proposals that reference this guidance.
 
+This is v0.2 of this document.
+
+### Framing
+
+For a security capability to be listed, it should:
+
+- Be successfully deployed in at least one ecosystem. We encourage individual package repositories to explore ideas, but we want to see results from production before recommending it broadly.
+
+- Not be something already widely deployed (like "use HTTPS"). This document is to help package repositories assess gaps and plan roadmaps, and as such is forward-looking.
```

Comment:
Honestly, I think that forbidding the use of HTTP in the upload and download of artifacts should be considered a part of this. I've been leading a crusade against the use of HTTP in the Java ecosystem and finally got it almost entirely eliminated:

https://infosecwriteups.com/burn-it-with-fire-how-to-eliminate-an-industry-wide-supply-chain-vulnerability-12515516fb56

I don't think that we should just assume this is the case for all artifact servers, especially new ones